features/cellular/framework/AT/ATHandler.cpp

```diff
@@ -25,9 +25,11 @@
 #include "rtos/Thread.h"
 #endif
 #include "Kernel.h"
+#include "CellularUtil.h"
 
 using namespace mbed;
 using namespace events;
+using namespace mbed_cellular_util;
```

Comment:
where is this coming from? Why is it not mbed::cellular_util rather?

Reply:
It has been there like that when CellularUtil was introduced. Just using it as it was defined. Do I need to change it?

Reply:
OK, not in this PR, but I would definitely put it under mbed namespace if needed.